Examples/GenerativeAICLI/Sources/GenerateContent.swift

```diff
@@ -93,27 +121,71 @@ struct GenerateContent: AsyncParsableCommand {
         parts.append(.data(mimetype: mimeType, imageData))
       }
 
-      let input = [ModelContent(parts: parts)]
+      input = [ModelContent(parts: parts)]
+
+      repeat {
+        try await processFunctionCalls()
 
-      if isStreaming {
-        let contentStream = model.generateContentStream(input)
-        print("Generated Content <streaming>:")
-        for try await content in contentStream {
-          if let text = content.text {
-            print(text)
+        if isStreaming {
+          let contentStream = model.generateContentStream(input)
+          print("Generated Content <streaming>:")
+          for try await content in contentStream {
+            processResponseContent(content: content)
           }
+        } else {
+          // Unary generate content
+          let content = try await model.generateContent(input)
+          print("Generated Content:")
```

Comment:
Can there ever be Generated Content when there are still function calls? 

I see three blanks in the sample before it prints.

Reply:
What about skipping the `print` if there is no text part in the response?

Reply:
I thought this print was part of the library. Just realized it's in the CLI. It's fine there. Sorry about my confusion.